platform/app/pluginConfig.json

```diff
@@ -96,7 +96,7 @@
     },
     {
       "packageName": "dicom-microscopy-viewer",
-      "importPath": "/dicom-microscopy-viewer/dicomMicroscopyViewer.min.js",
+      "importPath": "dicom-microscopy-viewer/dicomMicroscopyViewer.min.js",
```

Comment:
This change by itself is sufficient to load the dicom microscopy viewer inside the publicURL path, without the change to writePluginImportsFile change.  However, the library itself can't handle being loaded at a relative path.